src/components/opengraph/OpenGraphPreview.tsx

```diff
@@ -0,0 +1,88 @@
+import Markdown from 'components/core/Markdown/Markdown';
+import unescape from 'lodash/unescape';
+
+type Props = {
+  title: string;
+  description: string;
+  imageUrls: string[];
+};
+
+export const OpenGraphPreview = ({ title, description, imageUrls }: Props) => (
+  <>
+    <div className="container">
+      <div className="gallery">
+        {imageUrls.map((url) => (
+          <div key={url} className="image" style={{ backgroundImage: `url("${url}")` }} />
+        ))}
+      </div>
+      <div className="byline">
+        <div className="username">{unescape(title)}</div>
```

Comment:
just a heads up that titles are optional for Collections so we might not always have a title here. 

that shouldn't affect too much, one place I could see that matter is the separator? not sure if we want to show the separator if we don't have a title. 
that's more of a design decision though so I can call it out on discord.

Reply:
since we're blocking on a design pass, I'll leave this for the design work